src/earn/saga.test.ts

```diff
@@ -209,7 +204,7 @@ describe('depositSubmitSaga', () => {
     deposit: {
       inAmount: {
         value: '100',
-        tokenId: networkConfig.aaveArbUsdcTokenId,
+        tokenId: `${NetworkId['arbitrum-sepolia']}:0x460b97bd498e1157530aeb3086301d5225b91216`,
```

Comment:
same as above, can we use the test/values constant, or maybe just make a `mockAaveArbUsdcTokenId` alongside address

Reply:
`mockAaveArbUsdcTokenId` already existed so just replaced all uses like the example above with that!